lib/server.js

```diff
@@ -237,6 +237,13 @@ CoAPServer.prototype.listen = function(port, address, done) {
   this._port = port
   this._address = address
 
+  if (parameters.pruneTimerPeriod) {
+    // Start LRU pruning timer
+    this._lru.pruneTimer = setInterval(function () {
+      that._lru.prune()
+    }, parameters.pruneTimerPeriod*1000)
```

Comment:
it is below - one of the mock timer libraries doesn't support `unref()` so I had to put a guard around it.